eng/automation/utils.py

```diff
@@ -454,3 +454,21 @@ def compare_version(v1: str, v2: str) -> int:
     if v1_patch_version > v2_patch_version:
         return 1
     return 0
+
+
+def remove_customization_config(tsp_dir: str):
+    tsp_file_path = os.path.join(tsp_dir, "tspconfig.yaml")
```

Comment:
Is updating the tspconfig.yaml file the only way to do this? Can we not pass a property to tsp-client that gets passed to tsp compile to override this value?

Reply:
tsp-client fixed with 0.25.0